.vscode/launch.json

```diff
@@ -203,6 +203,29 @@
       "preLaunchTask": "Compile",
       "internalConsoleOptions": "openOnSessionStart"
     },
+    {
```

Comment:
This launch configuration is not needed.

Reply:
Thanks, I forgot about this change. I'll remove it.